### DISCUSSION TITLE
[MHRise] Overrides Flawless Widescreen

### DISCUSSION THREAD

**Discussion by Smiffleblurf** [praydog/REFramework#283] [General Questions] [answered]
Hi! Been using the Flawless Widescreen mod](https://www.flawlesswidescreen.org/) to be able to play Monster Hunter Rise on my 32:9 ultrawide 5120x1440 monitor.

After setting that up, I started noticing every two games MHRise crashes consistently. It was noted in the community using REFramework prevents this kind of crash. However, after dropping REFramework's .dll into MHRise's directory it appears to override the changes Flawless Widescreen made as it drops me back to a 21:9 aspect ratio.

Is there something I can do to keep REFramework from overriding Flawless Widescreen? Thanks!

**Comment by KrisCris** [comment, marked as answer]
https://www.nexusmods.com/monsterhunterrise/mods/40
> - aspect_ratio_fit: forces the game to always "fit" the image to the size of the screen (for 32:9 and others)

why not use this instead

**Reply by Smiffleblurf** [reply]
Thanks! I tried this method a while back and it does work out better for 32:9.
